src/Integration/RequestIntegration.php

```diff
@@ -41,18 +41,27 @@ final class RequestIntegration implements IntegrationInterface
      */
     private $options;
 
+    /**
+     * Fetches a PSR-7 request object for the current request.
```

Comment:
Please move this description on the same line as the `@var` tag (if you know another PHPCS rule to enforce it feel free to add it to the config). Also, this description is describing what the instance of the property is doing rather than what this property is, so I suggest to change it to something like `PSR-7 request fetcher`